packages/taiko-client/prover/proof_producer/sgx_producer.go

```diff
@@ -117,13 +129,155 @@ func (s *SGXProofProducer) RequestProof(
 	}, nil
 }
 
+func (s *SGXProofProducer) Aggregate(
+	ctx context.Context,
+	items []*ProofWithHeader,
+	requestAt time.Time,
+) (*BatchProofs, error) {
+	log.Info(
+		"Aggregate sgx batch proofs from raiko-host service",
+		"proofs", items,
+	)
+	if len(items) == 0 {
+		return nil, errors.New("invalid items length")
```

Comment:
If this `errors.New("invalid items length")` error is used multiple times, lets define it at somewhere at first?

Reply:
OK